web-ui/src/components/volunteer/VolunteerBadges.jsx

```diff
@@ -0,0 +1,104 @@
+import PropTypes from 'prop-types';
```

Comment:
This UI component renders information about volunteering activities for a specific member selected on the DIRECTORY ... People page.